static/script/devices/mediaplayer/html5.js

```diff
@@ -587,7 +587,7 @@ require.def(
                         self._mediaElement.currentTime = self._sentinelSeekTime;
                     });
                 } else {
-                    this._sentinelSeekTime = currentTime;
+                    this._sentinelSeekTime = undefined;
```

Comment:
As you say, David, I'm sure there was a reason for this. Why is this change needed for this pull request? :-)